common/test/src/TestUtils.h

```diff
@@ -122,4 +123,55 @@ namespace TrenchBroom {
     };
 
     GlobMatcher MatchesGlob(const std::string& glob);
+
+    template <typename T, std::size_t S>
+    class UnorderedApproxVecMatcher : public Catch::MatcherBase<std::vector<vm::vec<T,S>>> {
+    private:
+        std::vector<vm::vec<T,S>> m_expected;
+        T m_epsilon;
+    public:
+        explicit UnorderedApproxVecMatcher(const std::vector<vm::vec<T,S>>& expected, const T epsilon) :
```

Comment:
Shame that `Catch::UnorderedEquals` requires both vectors to have the same element type. It would be enough to require them to be comparable. Then we could use `vm::approx`.